docs/2021/minerva/updates/2021-08-17.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Week 10
+author: Shreya Singh
+---
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2021 Shreya Singh <shreya.out@gmail.com>
+-->
+# Coding Week-10 Meeting
+### Attendees
+
+- [Gaurav Mishra ](https://github.com/GMishx)
+- [Anupam Ghosh ](https://github.com/ag4ums)
+- [Michael C. Jaeger](https://github.com/mcjaeger)
+- [Shaheem Azmal ](https://github.com/shaheemazmalmmd)
+- [Ayush Bhardwaj ](https://github.com/hastagAB)
+- [Vasudev Maduri ](https://github.com/vasudevmaduri)
+- [Omar Mohamed ](https://github.com/OmarAbdelSamea)
+- [Kaushlendra Pratap ](https://github.com/Kaushl2208)
+- [Shreya Singh ](https://github.com/SinghShreya05)
+
+
+### Discussions
+1. To fix filepaths, remove duplicate scripts and hardcoded paths, update Readme and weekly reports after which the data generation repository could be integrated to Fossology
+2. Push all the licenses generated to repo and test different appraches using https://github.com/fossology/FOSSologyML/
+3. Integrating the Repo under Fossology.
+
+
+### Week-10 Progress
+1. Fixed the [Minerva-Dataset](https://github.com/SinghShreya05/Minerva-Dataset-Generation) filepaths, imports, duplicated codes.

```

Comment:
Please update the repo path to https://github.com/fossology/Minerva-Dataset-Generation